extensions/wikia/AdEngine/js/context/uapContext.js

```diff
@@ -1,8 +1,9 @@
 /*global define*/
 define('ext.wikia.adEngine.context.uapContext', [
+	'ad-engine.bridge',
```

Comment:
are you sure it won't break on mobile?

Reply:
It shouldn't right now, I added it also to mobile module

Reply:
hmmm, I think adding it as optional wouldn't bother us, but it's your call, I'm fine with both